orcestra_book/data_policy.md

```diff
@@ -0,0 +1,11 @@
+# Data Policy
+
+ORCESTRA encourages free use and access to data. We trust in the integrity of the scientific community to honor the contributions of those who collected these data.
+
+The ORCESTRA data are a product of the collective efforts and funding of various independent research groups. It often represents years of work in developing, certifying, and operating instruments, often tied to specific research or Ph.D. projects. We ask users to respect the priority of the data collectors by coordinating use with the instrument Principal Investigators (PIs) and by clearly acknowledging the source of the data in any publications or presentations.
+
+For publications using ORCESTRA data, please include this acknowledgement:
+
+> "The data used in this publication was collected during the ORCESTRA field campaign and is available through [insert institute name]. ORCESTRA is supported by the European Space Agency (ESA), the Max Planck Society (MPG), the German Research Foundation (DFG), the German Aerospace Center (DLR), the German Federal Ministry for Education and Research (BMBF), and the US National Science Foundation (NSF)."
```

Comment:
```suggestion
```

Reply:
With the suggested changes, I would have no idea what to put into the acknowledgements 🤷‍♂️

Reply:
How about

```suggestion
> "The data used in this publication was collected during the ORCESTRA field campaign and is available through [insert institute name]. ORCESTRA is an international initiative of institutes within Europe, North America, Barbados and Cape Verde.  [Add an additional sentence in case of specific fundings in the context of the use of specific data products]."
```